development-defaults.ini

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Multi-line strings cannot be specified as defaults in config specfiles, so I had to specify this here.